sdk/ai/azure-ai-projects/azure/ai/projects/models/_models.py

```diff
@@ -5613,6 +5673,106 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, type=VectorStoreChunkingStrategyResponseType.OTHER, **kwargs)
 
 
+class VectorStoreConfiguration(_model_base.Model):
+    """Vector storage configuration is the list of data sources, used when multiple
+    files can be used for the enterprise file search.
+
+
+    :ivar data_sources: Data sources. Required.
+    :vartype data_sources: list[~azure.ai.projects.models.VectorStoreDataSource]
+    """
+
+    data_sources: List["_models.VectorStoreDataSource"] = rest_field()
+    """Data sources. Required."""
+
+    @overload
+    def __init__(
+        self,
+        *,
+        data_sources: List["_models.VectorStoreDataSource"],
+    ) -> None: ...
+
+    @overload
+    def __init__(self, mapping: Mapping[str, Any]) -> None:
+        """
+        :param mapping: raw JSON to initialize the model.
+        :type mapping: Mapping[str, Any]
+        """
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+
+
+class VectorStoreConfigurations(_model_base.Model):
+    """The structure, containing the list of vector storage configurations i.e. the list of azure
+    asset IDs.
+
+
+    :ivar store_name: Name. Required.
+    :vartype store_name: str
+    :ivar store_configuration: Configurations. Required.
+    :vartype store_configuration: ~azure.ai.projects.models.VectorStoreConfiguration
+    """
+
+    store_name: str = rest_field(name="name")
+    """Name. Required."""
+    store_configuration: "_models.VectorStoreConfiguration" = rest_field(name="configuration")
+    """Configurations. Required."""
+
+    @overload
+    def __init__(
+        self,
+        *,
+        store_name: str,
+        store_configuration: "_models.VectorStoreConfiguration",
+    ) -> None: ...
+
+    @overload
+    def __init__(self, mapping: Mapping[str, Any]) -> None:
+        """
+        :param mapping: raw JSON to initialize the model.
+        :type mapping: Mapping[str, Any]
+        """
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+
+
+class VectorStoreDataSource(_model_base.Model):
+    """The structure, containing Azure asset ID and the asset type of the file used as a data source
+    the enterprise file search.
+
+
+    :ivar storage_uri: Asset URI. Required.
+    :vartype storage_uri: str
+    :ivar asset_type: The asset type *. Required. Known values are: "uri_asset" and "id_asset".
+    :vartype asset_type: str or ~azure.ai.projects.models.VectorStoreDataSourceAssetType
+    """
+
+    storage_uri: str = rest_field(name="uri")
+    """Asset URI. Required."""
+    asset_type: Union[str, "_models.VectorStoreDataSourceAssetType"] = rest_field(name="type")
+    """The asset type *. Required. Known values are: \"uri_asset\" and \"id_asset\"."""
+
+    @overload
+    def __init__(
+        self,
+        *,
+        storage_uri: str,
```

Comment:
would it be better to call storage_uri as asset_identifier ? This parameter can then hold either a URI or an ID asset

Reply:
Fixed

Reply:
I still see storage_uri being used